nyx_client/nyx_client/client.py

```diff
@@ -187,7 +191,7 @@ def _nyx_post(
                 content_type="multipart/form-data" if multipart else "application/json", extra_headers=headers
             ),
         )
-        if resp.status_code == 400:
+        if resp.status_code >= 400:
```

Comment:
nyx_put below still has `== 400` - should the `log.warning` say anything else - e.g. `nyx POST failed: ...` ?

Reply:
Whooops left over debug, nice spot!